Code/Editor/Plugins/ComponentEntityEditorPlugin/SandboxIntegration.cpp

```diff
@@ -1116,18 +1116,29 @@ bool SandboxIntegrationManager::CanGoToSelectedEntitiesInViewports()
 
 bool SandboxIntegrationManager::CanGoToEntityOrChildren(const AZ::EntityId& entityId) const
 {
+    AZ::Entity* entity = nullptr;
+    AZ::ComponentApplicationBus::BroadcastResult(entity, &AZ::ComponentApplicationRequests::FindEntity, entityId);
+    if (entity == nullptr)
+    {
+        return false;
+    }
+
+    // Skip if this is a UI component; UI components use LyShine's "UI Canvas" and are not visible in the Editor viewport
+    const AZ::Component* uiElementComponent = entity->FindComponent(AZ::Uuid("{4A97D63E-CE7A-45B6-AAE4-102DB4334688}"));
```

Comment:
Consider using a named variable for that Uuid:
`constexpr AZ::Uuid uiCanvasUUID = AZ::Uuid("{4A97D63E-CE7A-45B6-AAE4-102DB4334688}"));`

Reply:
Thanks, fixed!